pkgs/development/libraries/gdal/default.nix

```
@@ -14,7 +14,9 @@
 , useHDF ? (!useMinimalFeatures)
 , useNetCDF ? (!useMinimalFeatures)
 , useArmadillo ? (!useMinimalFeatures)
+, useJava ? (!useMinimalFeatures)
```

I compared closure sizes of gdal built with Java and without and the one built with Java is only little bit bigger. 

```
nix-build -E 'with import ./. {}; gdal.override { useJava = false; }'

nix path-info -rsSh /nix/store/wr943gniijj42v46ym1n31pcqz5mixky-gdal-3.8.3

/nix/store/wr943gniijj42v46ym1n31pcqz5mixky-gdal-3.8.3                        	  43.2M	   1.3G
```
```
nix-build -E 'with import ./. {}; gdal.override { useJava = true; }'

nix path-info -rsSh /nix/store/03l8izqrs7ni4dbizv4cbainl2p3nk0k-gdal-3.8.3

/nix/store/03l8izqrs7ni4dbizv4cbainl2p3nk0k-gdal-3.8.3                        	  45.2M	   1.3G
```

But even with such a small difference, I suggest building without Java by default due to larger size of build dependencies (`ant` and `jdk`).

```suggestion
, useJava ? false
```